packages/manager/apps/web/client/app/domain/dashboard/domain.constant.js

```diff
@@ -2,6 +2,32 @@ export const ALERTS = {
   tabs: 'domain_alert_tabs',
 };
 
+export const RENEW_URL = {
+  DEFAULT: '/cgi-bin/order/renew.cgi?domainChooser=',
```

Comment:
This url is not valid.
Also I think we can extract the suffix from the url

```suggestion
  const RENEW_URL_SUFFIX = 'cgi-bin/order/renew.cgi?domainChooser=';
  export const RENEW_URL = {
  AU: `https://ca.ovh.com/au/${RENEW_URL_SUFFIX}`,
  CA: `https://ca.ovh.com/fr/${RENEW_URL_SUFFIX}`,
  CZ: `https://www.ovh.cz/${RENEW_URL_SUFFIX}`,
```